docs/user-guide/rules.md

```diff
@@ -73,6 +73,12 @@ Disallow overrides with these `no-overrides` rules.
 
 - [`declaration-block-no-shorthand-property-overrides`](../../lib/rules/declaration-block-no-shorthand-property-overrides/README.md): Disallow shorthand properties that override related longhand properties (Ⓡ & Ⓢ).
 
+### Unmatchable
+
+Disallow unmatchable things with these `no-unmatchable` rules.
```

Comment:
minor

"things" should be replaced

Reply:
Gotcha, what's a good replacement? I'm not the best at writing docs 😅

Reply:
Me neither. Wait for jeddy3.

Reply:
Funnily enough, we've used "thing(s)" since the beginning of [our docs](https://github.com/stylelint/stylelint/blob/main/docs/developer-guide/rules.md#define-the-rule)

> the [thing](http://apps.workflower.fi/vocabs/css/en) the rule applies to, e.g. at-rule

We haven't been able to think of anything better. If anyone does, let's roll it out across all the instances of things in another PR. At the moment, let's keep it as things in this PR to be consistent with the other sections in the doc.